test/basic.html

```diff
@@ -297,7 +297,7 @@
 
       var json = f.serialize();
       assert.equal(Object.keys(json).length, 2);
-      
```

Comment:
@cdata This is technically the test that was failing, `does not submit forms with invalid native elements`